### PR TITLE
Making signup button available only when application is active

### PR DESCRIPTION
**Pull request by anuruddhal**

### CODE REVIEW

**Comment by ravihansa3000** [comment]
Merged in 08433049e6fbf026287a6159bb4098c9b0e413b5 and b22bb362dc1aa3ff27c8389b526bbb579ff0ddc8

**Comment by ravihansa3000** [comment]
#464 is closed

File: components/org.apache.stratos.rest.endpoint/src/main/java/org/apache/stratos/rest/endpoint/util/converter/ObjectConverter.java [hunk 1]
```diff
@@ -1147,15 +1151,18 @@ public static ApplicationBean convertStubApplicationContextToApplicationDefiniti
...
+        Application application = ApplicationManager.getApplications().getApplication(applicationContext.getApplicationId());
         ApplicationBean applicationDefinition = new ApplicationBean();
         applicationDefinition.setApplicationId(applicationContext.getApplicationId());
         applicationDefinition.setAlias(applicationContext.getAlias());
         applicationDefinition.setMultiTenant(applicationContext.getMultiTenant());
         applicationDefinition.setName(applicationContext.getName());
         applicationDefinition.setDescription(applicationContext.getDescription());
-        applicationDefinition.setStatus(applicationContext.getStatus());
-
+        if (application.getStatus().name().equals(APPLICATION_STATUS_ACTIVE)) {
```
Comment (swgkg) [inline components/org.apache.stratos.rest.endpoint/src/main/java/org/apache/stratos/rest/endpoint/util/converter/ObjectConverter.java]: Can't we reuse the ApplicationStatus enum without declaring the additional constant of APPLICATION_STATUS_ACTIVE?

Comment (anuruddhal) [inline components/org.apache.stratos.rest.endpoint/src/main/java/org/apache/stratos/rest/endpoint/util/converter/ObjectConverter.java]: Fixed with commit 3d44e8de5e74669c6344e5b24d0589badd838840

**Comment by imesh** [comment]
It would be better to close pull requests once merged.
Thanks

**Pull request closed without merging**